applications/dashboard/scss/admin.scss

```diff
@@ -4,8 +4,10 @@
 //
 // @author    Becky Van Bussel <beckyvanbussel@gmail.com>
 //
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i');
 
-@import 'src/variables';
```

Comment:
What happens with this src/variables sheet? Is it going away since it's not being used anymore?

Reply:
I kept it there because style.css is using it for some reason. I want to be able to tweak the settings for either theme without affecting the other. Whether the _variables.scss can be removed is not the purview of this PR.